BridgeAppSDK/SBAUserWrapper.swift

```diff
@@ -65,11 +65,16 @@ public protocol SBAUserWrapper: class, SBBAuthManagerDelegateProtocol {
     var password: String? { get set }
     
     /**
-     * Gender is stored in the keychain
+     * Gender describes the characteristics that a society or culture delineates as masculine or feminine. Stored in the keychain.
      */
     var gender: HKBiologicalSex { get set }
     
     /**
+     * Biological sex refers to the biological sex of the person at birth. Stored in the keychain.
```

Comment:
"Sex" isn't actually any more medically meaningful than "gender" as it turns out (refer to my previous comments for elaboration).